tests/functional/comments-auth.js

```diff
@@ -126,7 +126,7 @@ define([
         .findByCssSelector('.js-issue-comment-button').click()
         .end()
         .sleep(500)
-        .findAllByCssSelector('.js-issue-comment:not(.wc-Comment--form)')
+        .findAllByCssSelector('.js-issue-comment')
```

Comment:
Thank you for updating all of these -- it's tedious work, but now they won't break. :sunglasses: